CRM/MembershipExtras/Upgrader.php

```diff
@@ -380,27 +399,35 @@ private function isMoreThanOneMonthOld($date = null) {
   }
 
   private function createManageInstallmentActivityTypes() {
-    civicrm_api3('OptionValue', 'create', [
-      'option_group_id' => 'activity_type',
-      'name' => 'update_payment_plan_next_period',
-      'label' => 'Update Payment Plan Next Period',
-      'api.OptionValue.create' => [
+    $optionValues = [
+      ['name' => 'update_payment_plan_next_period', 'label' => 'Update Payment Plan Next Period'],
+      ['name' => 'update_payment_plan_current_period', 'label' => 'Update Payment Plan Current Period'],
+    ];
+    foreach ($optionValues as $optionValue) {
+      $result = civicrm_api3('OptionValue', 'get', [
+        'sequential' => 1,
+        'name' => $optionValue['name'],
```

Comment:
add 'option_group_id' => 'activity_type', since there might be option values with same names across different option values.

Reply:
Right.